cardform/src/main/java/com/mercadolibre/android/cardform/data/model/response/FieldsSetting.kt

```diff
@@ -8,5 +8,6 @@ internal data class FieldsSetting (
     val hintMessage : String,
     val validationPattern : String,
     val validationMessage : String,
-    val mask: String
-)
\ No newline at end of file
+    val mask: String,
+    val autocomplete: Boolean? = null
```

Comment:
this field cannot be false by default?

Reply:
no, because this should represent the field that comes from backend and it's nullable as default (so we decided to have default behaviors per field when we map them)